src/main/java/mate/academy/Main.java

```diff
@@ -2,16 +2,27 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.exception.RegistrationException;
+import mate.academy.lib.Injector;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
+import mate.academy.model.User;
+import mate.academy.service.AuthenticationService;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector injector =
+            Injector.getInstance("mate.academy");
+    private static final String EMAIL = "email";
+    private static final String PASSWORD = "Password";
```

Comment:
Ensure that the email used for registration is unique. This is crucial to prevent duplicate user entries in the database.